src/v/pandaproxy/rest/handlers.cc

```diff
@@ -68,10 +68,10 @@ namespace {
 
 using server = ctx_server<proxy>;
 
-ss::shard_id consumer_shard(const kafka::group_id& g_id) {
-    auto hash = xxhash_64(g_id().data(), g_id().length());
-    return jump_consistent_hash(hash, ss::smp::count);
-}
+// ss::shard_id consumer_shard(const kafka::group_id& g_id) {
+//     auto hash = xxhash_64(g_id().data(), g_id().length());
+//     return jump_consistent_hash(hash, ss::smp::count);
+// }
```

Comment:
I think you need to leave the consumers alone, they already have their own client. Probably they'll need credentials passed through if authZ is required.